### DISCUSSION TITLE
On key release binding?

### DISCUSSION THREAD

**Discussion by Yusup-Badiev** [google/neuroglancer#497] [General]
Hello! For the key bindings in https://github.com/google/neuroglancer/blob/master/src/neuroglancer/ui/default_input_event_bindings.ts#L92 is there any way to add a keybinding for a key release like `'at_release:keyr': 'rotation clean up'`

**Comment by xgui3783** [comment]
I wonder if `at:keyup` and then, check the key that was released would work?
